packages/frontend/src/hooks/useEcharts.ts

```diff
@@ -55,7 +57,7 @@ const getAxisTypeFromField = (
             }
         }
     } else {
-        return 'category';
+        return 'value';
```

Comment:
@ZeRego does this mean table calculations are always of type `value` ?

We don't have any type information for this right?

Reply:
Correct. This is the best default until we allow for users to set a type for table calculations.

Reply:
makes sense!